ingestors/pypi_rss.go

```diff
@@ -0,0 +1,63 @@
+package ingestors
+
+import (
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/librariesio/depper/data"
+	"github.com/mmcdole/gofeed"
+)
+
+//const pyPiPackagesFeedUrl = "https://pypi.org/rss/packages.xml"
+const pyPiUpdatesFeedUrl = "https://pypi.org/rss/updates.xml"
```

Comment:
Packagist had both feeds as well, but I ditched the packages feed because new packages would show up in the new releases feed anyway. For instance, this Pypi package with 1 version just showed up in the above feed: https://pypi.org/project/nevopy/0.0.1/#history 

Should we just skip the new packages feed?

Reply:
Hmm I've been checking pretty regularly and haven't seen them in the updates feed, will poke some more.

Reply:
<item>
      <title>sshcmd added to PyPI</title>
      <link>https://pypi.org/project/sshcmd/</link>
      <guid>https://pypi.org/project/sshcmd/</guid>
      <description>ssh cmd</description>
      <author>xliang.lee@qq.com</author>
      <pubDate>Tue, 02 Feb 2021 10:27:12 GMT</pubDate>
    </item>
  </channel>
</rss>

JP:~/Code/tidelift/depper$ curl https://pypi.org/rss/updates.xml | grep sshcmd
=> nothin

Reply:
Are you positive packagist did a better job than pypi is doing here? Have you run through a couple examples and seen them in both feeds?

Reply:
> Are you positive packagist did a better job than pypi is doing here? Have you run through a couple examples and seen them in both feeds?

Packagist could possibly return new packages that don't have versions yet ([here's the query](https://github.com/composer/packagist/blob/dc36ab17a66f2af14f77dd1273b88c12d82f0f3b/src/Entity/PackageRepository.php#L504)), but I figured we don't really care about a package until it has releases. Wondering what you and @katzj think about that tho.

Reply:
But here's an example I just saw in pypi feeds:

* from `updates.xml`: `multiplespawner 0.0.0a0` (Tue, 02 Feb 2021 15:31:25 GMT)
* from `packages.xml`: `multiplespawner added to PyPI` (Tue, 02 Feb 2021 15:31:25 GMT)

Reply:
I also don't think there's a way to "Create a new project" on Pypi, so the only way to create one is to release it via twine, which would require you to have a version set anyway. 

All that said, probably can't hurt if you still wanna do the packages feed still 🤷

Reply:
I guess the question is can we _count on_ the updates feed having the versions from the new packages. I haven't been able to prove that to myself as yet.

Reply:
I think they're just pulling releases from all projects, ordered by release created_at: https://github.com/pypa/warehouse/blob/main/warehouse/rss/views.py#L66-L72